core/src/main/kotlin/org/rewedigital/katana/Component.kt

```diff
@@ -181,6 +197,18 @@ class Component(
     inline fun <reified T> injectNow(name: Any? = null) =
         context.injectNow<T>(name)
 
+    /**
+     * Injects requested dependency immediately or returns `null` if no binding could be found.
+     *
+     * @see inject
+     * @see injectNow
+     * @throws InstanceCreationException
+     *
+     * TODO: Rename to getOrNull()?
```

Comment:
+1 for the api name change to `get`, `getOrNull`, etc.

Reply:
Thanks for your feedback. The rename will not be part of this PR but I will further think about it.